eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/model/SubSnpNoHgvs.java

```diff
@@ -300,125 +244,18 @@ public void setAssemblyMatch(boolean assemblyMatch) {
     }
 
     public String getReferenceInForwardStrand() {
-        if (contigOrientation.equals(Orientation.REVERSE)) {
-            return getTrimmedAllele(calculateReverseComplement(reference));
-        } else {
-            return getTrimmedAllele(reference);
-        }
+        return this.dbSnpVariantAlleles.getReferenceInForwardStrand();
     }
 
-    /**
-     * Removes leading and trailing spaces. Replaces a dash allele with an empty string.
-     */
-    private static String getTrimmedAllele(String allele) {
-        if (allele == null) {
-            return "";
-        }
-        allele = allele.trim();
-        if (allele.equals("-")) {
-            return "";
-        }
-        return allele;
+    private List<String> getAllelesInForwardStrand() {
+        return dbSnpVariantAlleles.getAllelesInForwardStrand();
```

Comment:
The previous method has an explicit this reference but this does not.

Reply:
Removed the explicit "this" as is not necessary